main.c

```diff
@@ -268,8 +274,23 @@ int main(int argc, char *argv[]) {
             goto see_help;
         }
         if (script != NULL) {
-            printf("The script %s is created. Run the script to do the actual work please. Remember to run it when the OverlayFS is not mounted.\n", filename_template);
             fclose(script);
+            if (force)
+            {
+                printf("The script %s is created. Running the script now, as force is set to true.\n", filename_template);
+                char *command = malloc(strlen(filename_template) * sizeof(char) + 10);
```

Comment:
Thanks for the contribution. A few points here:
- What does the magic number 10 denote?
- Could you please also free the `command` at the end of the scope?

Reply:
10 is just a number for some more characters.
I added commit to free **command**.